docs/reference/changelog.rst

```diff
@@ -16,6 +16,19 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release-2.12.0:
+
+2.12.0 (2025-06-06)
+-------------------
+
+New features:
+
+- Add a ``FilesystemMount`` model and a ``filesystem_mount`` parameter to the
```

Comment:
We don't have docs for these? 🥺

Reply:
No.  Will this be a subsequent task, @upils?